actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -752,7 +752,9 @@ def apply_form_for_options!(object, options) # :nodoc:
       #   def labelled_form_with(**options, &block)
       #     form_with(**options.merge(builder: LabellingFormBuilder), &block)
       #   end
-      def form_with(model: nil, scope: nil, url: nil, format: nil, **options, &block)
+      def form_with(model: false, scope: nil, url: nil, format: nil, **options, &block)
+        raise ArgumentError, "Form model object is nil" if model.nil?
```

Comment:
Looking at other ArgumentErrors, it seems the messages are defined as "X cannot be Y" or "X should not be Y"
```suggestion
        raise ArgumentError, "The :model argument cannot be nil" if model.nil?
```

Reply:
Thanks for pointing that out @p8! I've updated the error message.